### PR TITLE
Seven things I want you to know about Fantomas

### PR DESCRIPTION
**Pull request by nojaf**
Hey @stackedsax, @c-rindi and @jgiannuzzi, I wanted to write down some frustrations with Fantomas I had in recent weeks and months. There are a couple of recurring themes that keep popping up in GitHub issues and as a form of therapy, I wished to address these items.

Could you review this prose, please? Fair warning, the tone might be overall a tad negative, but then again these are frustrations ;).
In point 6 I disclose some details about the current sponsorship agreement between Axxes and GR. Is this ok to have in there?

Many thanks for your time (again).

### CODE REVIEW

**Comment by stackedsax** [comment]
Hey there Florian,

I started doing my normal little light edits, but I thought we might
connect on some of the messaging before I start to nitpick the verbiage.

I understand where you're coming from and the rationale behind some of the
decisions you're asserting here.  However, I don't think this article
articulates that rationale or gives any sense that there may be more
flexibility in the future.  It feels like the kind of post that would turn
away would-be contributors which is exactly the opposite of what you want
right now.

Let's chat and see if we can't tease out some of the rationale behind this
blog post.  You start by being open and transparent about your feelings
about carrying the load for this project.  I think that's a fine place to
come from (even if the tone is a little morose :D), but I would this as the
basis for the other decisions you so forcefully assert throughout the
article.

Why, for example, are there only two styleguides?  Because it's hard enough
getting these two to work.  But that doesn't mean there could never be
other styleguides once we get Fantomas "walking" and then "running".  This
is precisely why Fantomas needs more contributors -- to get Fantomas to a
place where we can think about greater flexibility for everyone's needs.
Come!  Join us in this valiant quest!

In short, I think we should sit down with Caterina and rethink the approach
here.  I think there's a way to be assertive about your current development
choices without scaring away any would-be users or contributors.

Cheers,

-Alex

**Comment by c-rindi** [comment]
Florian,
I'm with Alex on this, and would love to talk to you more about it before you publish anything. 
Will reach out shortly.
Thanks for running it by us!
~C

**Comment by nojaf** [comment]
This is why I'm allowed to blog unsupervised 😅. I agree with the remarks of Alex, it will send the wrong message.
The overall ideas might be somewhat there but the style of bringing them will not hit the mark.
I'm really open to discuss this and if necessary, we scrap the entire idea and go back to the drawing board.
